src/module.ts

```diff
@@ -46,6 +52,19 @@ export interface ModuleOptions {
    * @default 'directus_refresh_token'
    */
   cookieNameRefreshToken?: string;
+
+  /**
+   * Where to redirect after logging in with an OAuth Provider.
+   * @type string
+   */
+  redirectOnLogin?: string;
```

Comment:
Maybe we should implement this in the `loginWithProvider` method, as param?

Reply:
Good shout.